config/environments/test.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.middleware.use RackSessionAccess::Middleware
```

Comment:
@drod1000 @njgheorghita Is my understanding correct that this is necessary to support testing of the various user roles? FYI @drod1000